monitoring_protected_audience_api_services.md

```diff
@@ -141,173 +202,504 @@ These are metrics tracked on the BFE B&A servers.
+
+If the configured metrics are empty, i.e. no metrics are configured, then all metrics, including all privacy sensitive and non-sensitive metrics are collected.
+
+
+### configuring-export-interval
+
+Export intervals for metrics with and without noise can be configured separately. The metrics without noise are aggregated purely for performance reasons, whereas the ones with noise can achieve a better signal to noise ratio with aggregation. Thus, we have separate configurations for each.
+
+`metric_export_interval_ms` configures the export interval of non-noised metrics. The default value is 60,000ms (1 min).
+
+`dp_export_interval_ms` configures the export interval of noised metric. The default value is 300,000ms (5 min).
+
+
+#### Example
+
+
+```
+mode: PROD
+metric_export_interval_ms: 100000
+dp_export_interval_ms:  100000
+```
+
+
+For further information, see the [opentelemetry.io page](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#metric-points).
+
+
+### noise-related-configuration
+
+We expose several knobs to tune the noise configurations so an ad tech can most optimally use the overall privacy budget. Below are the noising parameters that ad tech can configure for each Privacy Impacting metric.
+
+
+#### max_partitions_contributed
+
+Default value: 1
+
+The maximum number of partitions  to be reported per request, higher value results in more noise added.
+
+Example:
+
+
+```
+metric { name: \"m_0\" max_partitions_contributed: 2 }
+```
+
+[code link](https://github.com/privacysandbox/data-plane-shared-libraries/blob/main/src/telemetry/flag/config.proto#L63)
+
+#### lower_bound, upper_bound
+
+Value is set based on ad tech’s observation on the metric, which range should cover most raw data, larger range results in more noise added.
+
+Example:
+
+
+```
+metric { name: \"m_0\" lower_bound: 1 upper_bound: 2 }
+```
+
+[code link](https://github.com/privacysandbox/data-plane-shared-libraries/blob/main/src/telemetry/flag/config.proto#L66)
+
+#### privacy_budget_weight
+
+Default value: 1
+
+All Privacy Impacting metrics split total privacy budget based on their weight. i.e. privacy_budget = total_budget\*privacy_budget_weight /total_weight.
+
+Example:
+
+
+```
+metric { name: \"m_0\" privacy_budget_weight: 2 }
+```
+
+[code link](https://github.com/privacysandbox/data-plane-shared-libraries/blob/main/src/telemetry/flag/config.proto#L72)
+
+#### drop_noisy_values_probability
+
+Default value: 0.0
+
+The probability that noisy values will be turned to 0. Setting this to a higher value ensures that the noisy values will be turned to 0, but also means that some actual (non-noisy values) may also be turned to 0. Setting this to 1 means all values will be turned to 0 (eliminating all noise, but also eliminating all useful data). This value should be set within an open range of (0.0,1.0).
```

Comment:
i think we should say what noisy values means ... as written this one is a little unclear.